docs/ACCEPTANCE_TESTS.md

```diff
@@ -0,0 +1,97 @@
+# Rule acceptance tests 
+
+## Goal
```

Comment:
I think it would be helpful to explain "why" the acceptance tests are important here. Why don't we want to declare existing valid datasets invalid? What's the real-world impact?

People will be directed to this page if a test fails, and they will need to understand why this is something they need to be concerned about.

Reply:
The new description is good! I would move the "Goal" section to the top, though, and have a separate "Definitions" section under that.